carma_wm/src/CARMAWorldModel.cpp

```diff
@@ -232,7 +232,7 @@ class LaneletDowntrackPair
   }
 };
 
-std::vector<lanelet::ConstLanelet> CARMAWorldModel::getLaneletsBetween(double start, double end,
+std::vector<lanelet::ConstLanelet> CARMAWorldModel::getLaneletsBetween(double start, double end, bool bounds_inclusive,
```

Comment:
This argument should not be included before the existing default valued argument as that will introduce unknown behavior in other components already relying on that default argument. It would be better to add this as another default argument after the shortest_path so that the old API is not broken.

Reply:
Switched the order and reversed other components that were affected